python/amici/petab_objective.py

```diff
@@ -150,6 +160,15 @@ def simulate_petab(
 
     # Compute total llh
     llh = sum(rdata['llh'] for rdata in rdatas)
+    # Compute total sllh
```

Comment:
I guess this was not the case before either, but maybe one could specify whether or not to calculate sensis? In the latter case, the simulation in runAmiciSimulations would be cheaper, and this mapping here not be performed.

Reply:
The optional `solver` argument is expected to be provided and already setup to compute sensitivities, if sensitivities are desired. Could default to first order forward? Either makes sense to me (no default at least ensures the user defines how sensitivities should be computed). Before review, it defaulted to `None`, and was handled in `aggregate_sllh`. Now defaults to `None` here, and `aggregate_sllh` is only called if sensitivities were computed (so, an error is now raised there, as per your other suggestion).

Reply:
> The optional solver argument is expected to be provided and already setup to compute sensitivities, if sensitivities are desired.

Yes!